src/components/POLine/POLine.js

```diff
@@ -211,6 +211,12 @@ class POLine extends Component {
             )}
           </Col>
         </Row>
+        <Accordion
```

Comment:
Can this leave outside of `AccordionSet` ? and collapse expand works fine? As well as collapse/expand all?

Reply:
collapse/expand works well, collapse/expand all doesn't works, but as I understood this one must be open by default and my decision was exclude it from 'all' functionality